packages/css/src/plugins/__tests__/atomicify-rules.test.tsx

```diff
@@ -0,0 +1,300 @@
+import postcss from 'postcss';
+import whitespace from 'postcss-normalize-whitespace';
+import autoprefixer from 'autoprefixer';
+import { atomicifyRules } from '../atomicify-rules';
+
+const transform = (css: TemplateStringsArray) => {
+  const result = postcss([atomicifyRules(), whitespace, autoprefixer]).process(css[0], {
+    from: undefined,
+  });
+
+  return result.css;
+};
+
+describe('atomicify rules', () => {
+  it('should atomicify a single declaration', () => {
+    const actual = transform`
+      color: blue;
+    `;
+
+    expect(actual).toMatchInlineSnapshot(`"._1doq13q2{color:blue}"`);
+  });
+
+  it('should should atomicify multiple declarations', () => {
+    const actual = transform`
+      color: blue;
+      font-size: 12px;
+    `;
+
+    expect(actual).toMatchInlineSnapshot(`"._1doq13q2{color:blue}._36l61fwx{font-size:12px}"`);
+  });
+
+  it('should autoprefix atomic rules', () => {
+    process.env.BROWSERSLIST = 'Edge 16';
+
+    const result = transform`user-select: none;`;
+
+    expect(result).toMatchInlineSnapshot(`"._q4hxglyw{-ms-user-select:none;user-select:none}"`);
+  });
+
+  it('should autoprefix atomic rules with multiple selectors', () => {
+    process.env.BROWSERSLIST = 'Edge 16';
+
+    const result = transform`
+      :hover, :focus {
+        user-select: none;
+      }
+    `;
+
+    expect(result).toMatchInlineSnapshot(
+      `"._1tclglyw:hover, ._16wpglyw:focus{-ms-user-select:none;user-select:none}"`
+    );
+  });
+
+  it.only('should autoprefix atrule atomic rules', () => {
```

Comment:
`.only` 🙈

Reply:
busted :P 

i should turn the eslint rule on for this.